README.md

```diff
@@ -39,3 +39,192 @@ On scoreboad terminal run:
     $ export SOL_MACHINE_ID="666a3d6a9d194a23b90a24573558d2f4"
     $ export SOL_FLOW_MODULE_RESOLVER_CONFFILE=foosball/scoreboard/sol-flow.json
     $ sol-fbp-runner foosball/scoreboard/main.fbp
+
+## Factory monitor
+
+This demo exemplifies a factory floor monitor: several terminal
+that connect, using OIC, to several monitors. Each monitor outputs
+current temperature reading and a FAILURE state.
+Terminals usually show monitor temperature on a round robin fashion.
+However, if any monitor sets its FAILURE state to true, terminals
+will stop showing temperature and will show the failure status.
+When on failure state, one cau use available buttons to 1) Dismiss
+failure status 2) Report to some 'supervisor' - action that dismiss
+failure status as well. After dismiss, terminals go back to showing
+monitors temperature.
+
+### Testing on development machine
+
+To simplify testing, this demo provides a sol-flow.json that uses GTK
+widgets as input/output. It also provides a server-dummy.fbp that doesn't
+really reads a temperature sensor, but instead show fake increasing
+values. To use the server-dummy.fbp, search for `##Server nodes` on
```

Comment:
Shouldn't we call all server fbps as monitor, instead? That's clearer to me, given the introductory text.